media/js/firefox/whatsnew/whatsnew-115-na-win7.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+// Create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
```

Comment:
Nit: this doesn't seem to be needed?

Reply:
you're right, a leftover from a previous iteration.